plugins/modules/lxd_container.py

```diff
@@ -172,6 +173,16 @@
         required: false
         aliases: [ cert_file ]
         type: path
+    verify:
+        description:
+          - In the case where the certificate is self-signed (LXD's default),
+            you may opt to disable the TLS fingerprint verification with verify=False.
```

Comment:
```suggestion
            you may opt to disable the TLS certificate validation with I(verify=false).
```
This is likely not fingerprint validation, but certificate validation?

Reply:
* Perhaps pxLXD does [this](https://github.com/lxc/lxd/issues/6802#issuecomment-579756153) .   
* The documentation [suggests so](https://pylxd.readthedocs.io/en/latest/usage.html?highlight=fingerprint#client) .     

*What now ?*

Reply:
Digging through the code of pyLXD, requests, and urllib3 shows:
- It actually verifies certificate fingerprints instead of validating certificate chains, **but only when it finds the server certificate locally**.
- When a string is passed to `verify`, it will use that as a path to the certificate to read.
- If `verify=true` is passed, it looks in `${CERTS_PATH}/servercerts/${REMOTE}.crt` (where `CERTS_PATH` can have one of multiple values: https://github.com/lxc/pylxd/blob/72426bf7c5d2b8768cfd1779523c9860cfafd266/pylxd/client.py#L36-L42, and where `REMOTE` is extracted from the endpoint) - if that exists, it will use that one, if it does not exist, it will fall back to certificate chain validation.

So it's pretty complicated.

Reply:
* While I cannot guarantee, I have a feeling this should not break old playbooks.
* Security is better than what happenend in the old module, where all was silently ignored.
* pyLXD shows regular contributions from lxd team.
  * They watch and test [backward compatibility e.g.](https://github.com/lxc/pylxd/commit/1c442f007e65cf72b4fa36e5abec4604a8063317)

You had stomach ache with calling it a bug fix, since it adds the verify option.
If we :
* call it a feature pull request 
* use verify is a bool (default true)
  * make it win over server_cert when false
* have a 2nd  new parameter server_cert (default ${CERTS_PATH}/servercerts/${REMOTE}.crt)
  * describe that it also takes an absolute path for use with fingerprint verification

Do You think it can replace the module under the same name as the next version ?  
*See the functionality is practically going to break for everyone, if nothing happens*